library/src/main/java/com/getstream/sdk/chat/view/ChannelHeaderViewStyle.kt

```diff
@@ -0,0 +1,104 @@
+package com.getstream.sdk.chat.view
+
+import android.content.Context
+import android.graphics.Color
+import android.graphics.Typeface
+import android.graphics.drawable.Drawable
+import android.text.TextUtils
+import android.util.AttributeSet
+import com.getstream.sdk.chat.R
+import com.getstream.sdk.chat.style.TextStyle
+
+class ChannelHeaderViewStyle(context: Context?, attrs: AttributeSet?) : BaseStyle() {
+    var channelTitleText: TextStyle
+    var lastActiveText: TextStyle
+    val isBackButtonShow: Boolean
+    val isLastActiveShow: Boolean
+    val isAvatarGroupShow: Boolean
+    val isOptionsButtonShow: Boolean
+    val optionsButtonTextSize: Int
+    val optionsButtonWidth: Int
+    val optionsButtonHeight: Int
+    val isActiveBadgeShow: Boolean
+
+    internal val backButtonBackground: Drawable?
+    internal val optionsButtonBackground: Drawable?
+    private val offlineText: String?
+    private var channelWithoutNameText = ""
+
+    fun getChannelWithoutNameText(): String {
+        return if (!TextUtils.isEmpty(channelWithoutNameText)) channelWithoutNameText else context.getString(R.string.stream_channel_unknown_title)
```

Comment:
Because Kotlin provide some extension function over String, we could do something like
```suggestion
        return channelWithoutNameText.takeIf(String::isNotBlank) ?: context.getString(R.string.stream_channel_unknown_title)
```

Reply:
Agree, i still need to get used to this one, but i like it!